src/main/scala/main/solutions/MazeSolver/Point.scala

```diff
@@ -0,0 +1,5 @@
+package main.solutions.MazeSolver
+
+case class Point(x: Int, y: Int) {
```

Comment:
in case class you don'y need the curly braces. it's basically like a struct in C that gives you setters and getters. 
if you want to add some methods and stuff, it's a regular class.